python_modules/libraries/dagster-dbt/dagster_dbt/ops.py

```diff
@@ -103,4 +107,4 @@ def dbt_docs_generate_op(context):
     (dbt_seed_op, "seed"),
     (dbt_docs_generate_op, "docs generate"),
 ]:
-    op.__doc__ = _get_doc(op.name, cmd)
+    op.__doc__ = _get_doc(_op.name, cmd)
```

Comment:
this looks like it's setting the dagster.op docstring. I think instead this should be `for dbt_op, cmd ...` and `dbt_op.__doc__ = _get_doc(dbt_op.name, cmd)`

Reply:
good catch